ts/components/RegionServiceWebView.tsx

```diff
@@ -103,6 +104,12 @@ const RegionServiceWebView: React.FunctionComponent<Props> = (props: Props) => {
       props.handleWebMessage(event.nativeEvent.data);
     }
 
+    const urlParse = new URLParse(props.uri, true);
```

Comment:
```suggestion
    const urlParsed = new URLParse(props.uri, true);
```
this could be a const defined in the compoent scope.
So it can be reused in `Webview` `originWhitelist`